tox.ini

```diff
@@ -7,9 +7,8 @@
 envlist = py27, py3
 
 [testenv]
-commands = nose2 --with-coverage --coverage-report=annotate --coverage-report=term --coverage=mediatimestamp
+commands = python -m unittest discover -s tests
```

Comment:
Could you mention the test runner change in the changelog?

Reply:
Wilco.